autorest/codegen/models/base_builder.py

```diff
@@ -50,14 +52,23 @@ def __init__(
             overloads or []
         )
         self._summary: str = yaml_data.get("summary", "")
-        # for operations where we don't know what to do, we mark them as abstract so users implement
-        # in patch.py
-        self.abstract = abstract
         self.want_tracing = want_tracing
         self.group_name: str = yaml_data["groupName"]
         self.is_overload: bool = yaml_data["isOverload"]
         self.api_versions: List[str] = yaml_data["apiVersions"]
 
+        if code_model.options["version_tolerant"] and yaml_data.get("abstract"):
```

Comment:
basically moved abstract marking code to `m4reformatter`, and moved the warning to base builder. This centralizes our code more